ui-react/packages/atlasmap-ui/src/AtlasmapUI/AtlasmapContextToolbar.tsx

```diff
@@ -58,223 +58,211 @@ export const AtlasmapContextToolbar: FunctionComponent<IAtlasmapContextToolbarPr
+          }
+        >
+          <Button
+            variant={'plain'}
+            aria-label="Reset all"
+            onClick={onResetAtlasmap}
+          >
+            Reset all
+          </Button>
+        </Tooltip>
+      </ToolbarItem>
+      <ToolbarItem>
+        <Tooltip
+          position={'auto'}
+          enableFlip={true}
+          content={<div>Show/hide mapping preview</div>}
+        >
+          <Button
+            variant={'plain'}
+            aria-label="Show/hide mapping preview"
+            onClick={onToggleShowMappingPreview}
+          >
+            <EyeIcon />
+          </Button>
+        </Tooltip>
+      </ToolbarItem>
+      <ToolbarItem>
+        <Tooltip
+          position={'auto'}
+          enableFlip={true}
+          content={<div>Show/hide mapping table</div>}
+        >
+          <Button variant={'plain'} aria-label="Show/hide mapping table">
+            <TableIcon />
+          </Button>
+        </Tooltip>
+      </ToolbarItem>
+      <ToolbarItem>
+        <Tooltip
+          position={'auto'}
+          enableFlip={true}
+          content={<div>Show/hide namespace table</div>}
+        >
+          <Button variant={'plain'} aria-label="Show/hide namespace table">
+            <CodeIcon />
+          </Button>
+        </Tooltip>
+      </ToolbarItem>
+      <ToolbarItem>
+        <Tooltip
+          position={'auto'}
+          enableFlip={true}
+          content={<div>Show/hide types</div>}
+        >
+          <Button
+            variant={'plain'}
+            aria-label="Show/hide types"
+            onClick={onToggleShowTypes}
+          >
+            <InfoIcon />
+          </Button>
+        </Tooltip>
+      </ToolbarItem>
+      <ToolbarItem>
+        <Tooltip
+          position={'auto'}
+          enableFlip={true}
+          content={<div>Show/hide mapped fields</div>}
+        >
+          <Button
+            variant={'plain'}
+            aria-label="Show/hide mapped fields"
+            onClick={onToggleShowMappedFields}
+          >
+            <MapMarkedIcon />
+          </Button>
+        </Tooltip>
+      </ToolbarItem>
+      <ToolbarItem>
+        <Tooltip
+          position={'auto'}
+          enableFlip={true}
+          content={<div>Show/hide unmapped fields</div>}
+        >
+          <Button
+            variant={'plain'}
+            aria-label="Show/hide unmapped fields"
+            onClick={onToggleShowUnmappedFields}
+          >
+            <MapIcon />
+          </Button>
+        </Tooltip>
+      </ToolbarItem>
+      <ToolbarItem>
+        <Tooltip
+          position={'auto'}
+          enableFlip={true}
+          content={<div>Show/hide free view</div>}
+        >
+          <Button
+            variant={'plain'}
+            aria-label="Show/hide free view"
+            onClick={onToggleShowFreeView}
+          >
+            <PficonDragdropIcon />
+          </Button>
+        </Tooltip>
+      </ToolbarItem>
+      {showFreeView && (
```

Comment:
I missed when the zoom buttons have been moved to the context toolbar, but I think their place is the control bar. I mean, it's exactly how the Topology component is showcased [in the doc](https://www.patternfly.org/v4/documentation/react/components/topology).

Reply:
I understand, but this was a direct request from the UX team.

Reply:
Maybe they missed out that detail, wdyt?

Reply:
I doubt it. If you look at the JIRA above, they have a screenshot that specifically points to how it was laid out before, and how they want it moved. We could pull them into the conversation if you like.

Reply:
Yeah sure. I mean we didn’t have the zoom buttons before and they make sense inside the canvas as per documentation. The other buttons are ok in the top toolbar